File: stanlib-tech/Models/Product.cs
namespace stanlib_tech.Models
{
    public class Product
    {
        public int Id { get; set; }

        public decimal SalePrice { get; set; }

        public string? Category { get; set; }
        public string? Description { get; set; }

        public string? Image { get; set; }
    }
}

File: stanlib-tech-ui/src/app/models/product.ts

export interface Product {
    id: number;
    description: string;
    salePrice: number;
    category: string;
    image: string;
}

export interface ProductSale {
    saleId: number;
    productId:number;
    saleQty: number;
    salePrice: number;
    saleDate: Date;
}